linux_docker_resources/Dockerfile

```
@@ -108,7 +108,7 @@ ADD rticonnextdds-license/rti_license.dat /tmp/rti_license.dat
 RUN apt-get update && apt-get install --no-install-recommends -y libasio-dev libssl-dev libtinyxml2-dev valgrind
 
 # Install the CycloneDDS dependencies.
-RUN apt-get update && apt-get install --no-install-recommends -y libcunit1-dev maven openjdk-11-jdk
+RUN apt-get update && apt-get install --no-install-recommends -y libcunit1-dev maven openjdk-11-jdk bison
```

```suggestion
RUN apt-get update && apt-get install --no-install-recommends -y bison libcunit1-dev maven openjdk-11-jdk
```

Let's preserve the lexical sorting.